conf/app.routes

```diff
@@ -2,6 +2,4 @@
 
 POST        /:empRef/process-file               controllers.DataUploadController.processFileDataFromFrontend(empRef:String)
 
-POST        /:empRef/process-csv-file           controllers.DataUploadController.processCsvFileDataFromFrontend(empRef:String)
```

Comment:
It appears only the V2 one is referenced.  The `processCsvFileDataFromFrontend` method on the controller was marked for removal so I have also removed the route that references it.
I would be happy to remove this commit if you think deleting the code is too high risk, though.

Reply:
if you started this, it can be competed, but needs more changes (or you can revert this change, up to you)
this endpoint is referenced in ers-returns-frontend here:
here: https://github.com/hmrc/ers-returns-frontend/blob/b5837353b8866fbc6fc416745725d2200ee74530/app/connectors/ErsConnector.scala#L141
so this needs changes, also feature toggle removing from this repository and configs

Reply:
PRs are now in place for the following repos in light of this change:

* ers-returns-frontend
* app-config-base
* app-config-staging
* app-config-production